spa-react/src/hooks/use-getUsers.hook.ts

```diff
@@ -0,0 +1,34 @@
+import { collection, getDocs } from "firebase/firestore";
+import { useState } from "react";
+import { db } from "src/utils/firebase";
+
+export interface IUser {
+  name: string;
+  surname: string;
+  id: string;
+  date: string;
+}
+
+export const useGetUsersFromDB = () => {
+  const [usersFromDB, setUsersFromDB] = useState<IUser[]>();
+
+  const getUsers = () => {
+    const usersCollectionRef = collection(db, "usersDB");
+    getDocs(usersCollectionRef)
+      .then((response) => {
+        const users = response.docs.map((doc) => ({
```

Comment:
Are you sure that docs exist?

Reply:
if there is no such doc, firestore will create it. I checked it, so we dont need to write condition for checking the existence of the doc

Reply:
If you don't have any docs you will have have a crash in doc.data()